themes/custom/droopler_theme/scss/components/_d_p_counters.scss

```diff
@@ -9,6 +9,7 @@ $d-p-counters-line-color-inverted: $line-background !default;
 $d-p-counters-header-color: $color-odysseus !default;
 $d-p-counters-number-color: $color-odysseus !default;
 $d-p-counters-text-color: $color-nestor !default;
+$d-p-counters-long-text-color: $color-odysseus !default;
```

Comment:
Wrzućmy to od razu do do https://github.com/droptica/droopler/blob/master/themes/custom/droopler_theme/STARTERKIT_SCSS/scss/config/_base_theme_overrides.scss